docs/specs/oidc.md

```diff
@@ -1,57 +1,65 @@
-# OIDC
-
-Authgear acts as OpenID Provider (OP).
-
+- [OIDC](#oidc)
   * [OAuth 2 and OIDC Conformance](#oauth-2-and-oidc-conformance)
   * [Client Metadata](#client-metadata)
-    * [Standard Client Metadata](#standard-client-metadata)
-    * [Custom Client Metadata](#custom-client-metadata)
-      * [Generic RP Client Metadata example](#generic-rp-client-metadata-example)
-      * [Native application Client Metadata example](#native-application-client-metadata-example)
-      * [Web application sharing the same root domain Client Metadata example](#web-application-sharing-the-same-root-domain-client-metadata-example)
-      * [Silent Authentication Client Metadata example](#silent-authentication-client-metadata-example)
+    + [Standard Client Metadata](#standard-client-metadata)
+    + [Custom Client Metadata](#custom-client-metadata)
+      - [Generic RP Client Metadata example](#generic-rp-client-metadata-example)
+      - [Native application Client Metadata example](#native-application-client-metadata-example)
+      - [Web application sharing the same root domain Client Metadata example](#web-application-sharing-the-same-root-domain-client-metadata-example)
+      - [Silent Authentication Client Metadata example](#silent-authentication-client-metadata-example)
   * [Authentication Request](#authentication-request)
-    * [scope](#scope)
-    * [response_type](#response_type)
-    * [display](#display)
-    * [prompt](#prompt)
-    * [max_age](#max_age)
-    * [id_token_hint](#id_token_hint)
-    * [login_hint](#login_hint)
-    * [acr_values](#acr_values)
-    * [code_challenge_method](#code_challenge_method)
+    + [scope](#scope)
+    + [response_type](#response-type)
+    + [prompt](#prompt)
+    + [max_age](#max-age)
+    + [id_token_hint](#id-token-hint)
+    + [login_hint](#login-hint)
+    + [acr_values](#acr-values)
```

Comment:
Seems the anchors are wrong after updating?

Reply:
I used https://ecotrust-canada.github.io/markdown-toc/ to generate the TOC, but it is buggy as you see. Now I use https://github.com/jonschlinkert/markdown-toc